README.md

```diff
@@ -1,6 +1,9 @@
+![CI](https://github.com/LSSTDESC/skyCatalogs/actions/workflows/ci.yml/badge.svg?branch=main)
+[![codecov](https://codecov.io/gh/LSSTDESC/skyCatalogs/branch/main/graph/badge.svg?branch=main)](https://codecov.io/gh/LSSTDESC/skyCatalogs)
+
```

Comment:
One of the admins likely need to add a CODECOV_TOKEN to the secrets before this can work, but that's not urgent.